platforms/documentation/docs/src/docs/userguide/reference/dependency-management/declaring-dependencies/dependency_constraints.adoc

```diff
@@ -81,6 +81,94 @@ include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencie
 include::sample[dir="snippets/dependencyManagement/managingTransitiveDependencies-versionsWithConstraints/groovy",files="build.gradle[tags=dependency-strictly]"]
 ====
 
+[[sec:strictly-constraint-precedence]]
+== Behavior of `strictly` Constraints at Different Dependency Graph Depths
+
+When multiple `strictly` version constraints are declared for the same dependency at different levels of the dependency graph, Gradle honors the constraint that is *closest to the root* of the graph and ignores constraints further from the root.
+
+This means that a `strictly` constraint declared in your project's direct dependencies will take precedence over `strictly` constraints declared in transitive dependencies or subprojects.
+
+.Root-level `strictly` constraint takes precedence
+====
+In this example, the root project declares a `strictly` constraint for version 4.4, while a subproject declares a different `strictly` constraint for version 4.5.
+The root-level constraint (4.4) takes precedence.
+
+[tabs]
```

Comment:
This needs to be turned into a tested snippet, you can see how to do that there: https://github.com/gradle/gradle/tree/master/platforms/documentation#code-snippets

There are many examples you can use. I recommend copy pasting an existing one and adjusting the build files to include the code below.